esm_parser/esm_parser.py

```diff
@@ -783,7 +789,8 @@ def add_entries_from_chapter(config, add_chapter, add_entries):
             for entry in my_entries:
                 config[add_chapter].append(entry)
         elif type(config[add_chapter]) == dict:
-            dict_merge(config[add_chapter], add_entries)
+            # MA: I'm not supper happy about the resolve_nested_adds implementation
+            dict_merge(config[add_chapter], add_entries, resolve_nested_adds=True)
```

Comment:
So, it's only false on the very first pass? Am I reading that right?

Reply:
Yes, you are reading that right because I did not add the resolve_nested_adds inside dict_merge, which is dumb... resolving add should be down recursively down the nesting, and not only on the first one. I'll fix that in the next commit.

Reply:
Commit 5d16a94 should solve this concern so that something that looks like, if I understood your question correctly.